content/5_Plat/Segtree_Ext.mdx

```diff
@@ -111,6 +116,160 @@ int main() {
 }
 ```
 
+</CPPSection>
+</LanguageSection>
+
+<FocusProblem problem="walkSam2" />
+
+### Solution - Ordered Set
+
+First, we coordinate compress all the values, and keep track of the frequency of each value in the array we build our segment tree on.
+We can answer the
+$\texttt{INSERT}$, $\texttt{DELETE}$, and $\texttt{COUNT}$ queries using
+the sum segment tree from the [PURS](/gold/PURS) module. Answering the
+$\texttt{K-TH}$ queries in $\mathcal{O}(\log{N})$ time requires us to use
+segment tree walking.
+
+Let $freq[i]$ equal the number of times $i$ occurs in our set.
```

Comment:
you should texttt variable names > 1 char

Reply:
oh shoot i forgot to address ur changes LOL mb